node/pkg/watchers/algorand/watcher.go

```diff
@@ -186,7 +186,6 @@ func (e *Watcher) Run(ctx context.Context) error {
 
 	// Create an error channel
 	errC := make(chan error)
-	defer close(errC)
```

Comment:
🤔 I feel like not closing the channel is introducing a code smell. Perhaps deleting this line just masks the bug instead of fixing it?

Reply:
what was the panic?